holoviews/tests/util/__init__.py

```diff
@@ -121,9 +121,38 @@ def test_cell_opts_util_plot(self):
     def test_cell_opts_util_norm(self):
         mat1 = hv.Image(np.random.rand(5,5), name='mat1')
         self.assertEqual(mat1.id, None)
-        opts("Image {+axiswise}", mat1)
+        opts.cellmagic("Image {+axiswise}", mat1)
```

Comment:
Bit confused by these, isn't this ``opts._cellmagic`` now?

Reply:
I guess those tests weren't being run!